bin/fit_posparams

```diff
@@ -0,0 +1,383 @@
+#!/usr/bin/env python
+# -*- coding: utf-8 -*-
+"""
+Analysis script comparing internally-tracked (theta, phi) to fvc-measured (x,y).
+The goal is to examine the general performance of all positioners during any
+move (not just a specialized calibration sequence).
+
+Data sources:
+
+    theta and phi:
+    --------------
+    The positioner moves database contains time-stamped data for values POS_T
+    and POS_P. These correspond to focalplane coordinates posintTP.
+    
+    x and y:
+    --------
+    The NERSC and KPNO servers contain numerous FITS files. These are processed
+    by desimeter. A result X_FP, Y_FP is generated from the centroids. This
+    corresponds to focalplane coordinates ptlXY.
+    
+    As of 2020-04-08, J. Guy identified 9472 such files from the commissioning
+    operations so far at the Mayall, and has run them through desimeter. See
+    Julien's email to desi-commiss@desi.lbl.gov of 2020-04-08, in which he
+    attaches several descriptive slides.
+
+    The fvc catalogs he produced are on NERSC at:
+        /global/cfs/cdirs/desi/engineering/fvc/desimeter/v0.2.0/
+
+    And the matched coordinates per positioner are on NERSC at:
+        /global/cfs/cdirs/desi/engineering/fvc/desimeter/v0.2.0/positioners/
+"""
+
+import os
+import sys
+import cProfile, pstats
+import pandas
```

Comment:
So far we have avoided a pandas dependency in the offline code to simplify the number of moving parts, preferring to use vanilla numpy structured arrays or astropy Tables (which aren't necessarily better than Pandas, but just picking that one since we already have an astropy dependency for other reasons).  i.e. can we avoid bringing in pandas as a desimeter dependency?

Reply:
Sure, I'll switch to astropy